src/components/templates/pinned.tsx

```diff
@@ -12,25 +12,37 @@ import {Heading} from 'components/atoms/heading';
 
 interface LayoutProps {
   heading?: string;
+  containerStyle?: ViewStyle;
```

Comment:
What's the reason to change a template?

Reply:
@floridemai  It's to get the app version pinned to the bottom of the screen. I tried to avoid changing the template, but:

- Without moving the horizontal padding, the card shadows get cropped by the edge of the inner ScrollView.
- Without tweaking the template, we'd have to add a duplicate achieving the same thing slightly differently for Settings only.

It looks like a lot in the diff but it basically just moves the horizontal padding to an inner view and adds a couple of optional style props. I checked the other screens that use `PinnedBottom` and they seem to be unchanged.

Reply:
@floridemai I'm merging this based on Shaun's approval for the early build, ping me if you want me to revisit it